From: Andreas Stieger <andreas.stieger@gmx.de>
Date: Sun, 25 Jan 2015 00:23:57 +0100
Subject: remove Debian specific command
Upstream: no

Remove Debian specific architecture detection and replace with build time setting.

---
 eatmydata.in |    4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

Index: libeatmydata-105/eatmydata.in
===================================================================
--- libeatmydata-105.orig/eatmydata.in
+++ libeatmydata-105/eatmydata.in
@@ -15,9 +15,7 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-export `dpkg-architecture|grep DEB_BUILD_MULTIARCH`
-
-shlib="/usr/lib/$DEB_BUILD_MULTIARCH/eatmydata.sh"
+shlib="@libexecdir@/eatmydata.sh"
 if [ -f "$shlib" ]; then
     . "$shlib"
 else
